core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -99,6 +100,9 @@ class ControllerApis(val requestChannel: RequestChannel,
         case ApiKeys.SASL_HANDSHAKE => handleSaslHandshakeRequest(request)
         case ApiKeys.SASL_AUTHENTICATE => handleSaslAuthenticateRequest(request)
         case ApiKeys.CREATE_PARTITIONS => handleCreatePartitions(request)
+        case ApiKeys.DESCRIBE_ACLS => aclApis.handleDescribeAcls(request)
```

Comment:
When would the controller ever process `DescribeAcls`?  It seems to me that it never would because brokers don't forward that request.  And if that is correct, I think my change to `DescribeAclsRequest.json` to set `"listeners": ["zkBroker", "broker", "controller"],` should instead be setting it to `"listeners": ["zkBroker", "broker"],`.

Reply:
We might want the controller to process DescribeAcls for debug purposes.  There's no reason to artificially disable it from processing the RPC, although I agree that it will normally not be used.